pkg/e2e/operators/ocmagent.go

```diff
@@ -0,0 +1,37 @@
+package operators
+
+import (
+	"github.com/onsi/ginkgo/v2"
+	"github.com/openshift/osde2e/pkg/common/alert"
+	"github.com/openshift/osde2e/pkg/common/helper"
+)
+
+var (
+	ocmAgentTestPrefix = "[Suite: operators] [OSD] OCM Agent Operator"
```

Comment:
Leave this in informing suite until we are in production.

```suggestion
	ocmAgentTestPrefix = "[Suite: informing] [OSD] OCM Agent Operator"
```